### PR TITLE
4.4 nancompare

### PR DESCRIPTION
**Pull request by lidiazuin**
For some reason the changes in LoadCSVTest.scala weren't applied locally to me, but this is already solved [here](https://github.com/neo4j/neo4j-documentation/pull/1504), so no need to review.

Please just check what's been changed in comparison.asciidoc.

### CODE REVIEW

**Review by loveleif — APPROVED** [review]
Thanks! Don't forget 5.0 also.

File: cypher/cypher-docs/src/docs/dev/syntax/comparison.asciidoc [hunk 1]
```diff
@@ -30,7 +30,7 @@ The comparison operators `\<=`, `<` (for ascending) and `>=`, `>` (for descendin
 The following points give some details on how the comparison is performed.
 
 * Numerical values are compared for ordering using numerical order (e.g. `3 < 4` is true).
-* The special value `java.lang.Double.NaN` is regarded as being larger than all other numbers.
+* All comparability tests (<, <=, >, >=) with `java.lang.Double.NaN` evaluate as false. For exemple, `1 > b` and `1 < b` are both false when b is NaN.
```
Comment (davidoliverSP2) [inline cypher/cypher-docs/src/docs/dev/syntax/comparison.asciidoc]:
```suggestion
* All comparability tests (<, <=, >, >=) with `java.lang.Double.NaN` evaluate as false. 
For example, `1 > b` and `1 < b` are both false when b is NaN.
```

**Review by davidoliverSP2 — APPROVED** [review]
Thanks @lidiazuin 

One small editorial suggestion

File: cypher/cypher-docs/src/docs/dev/syntax/comparison.asciidoc (hunk 1 above)
Comment (NataliaIvakina) [inline cypher/cypher-docs/src/docs/dev/syntax/comparison.asciidoc]: tiny typo: 'exemple' should be 'example'

**Pull request merged**